File: C/Program44.c
//////////////////////////////////////////////////////////////////////////////////////////////////////
//                                                                                                  //
//  File name :     Program44.c                                                                     //
//  Description :   Program to accept n integers from user & return Count of Even Number            //
//  Author :        Pallavi Omprakash Malewar                                                       //
//  Date :          27/06/2025                                                                      //
//                                                                                                  //
//////////////////////////////////////////////////////////////////////////////////////////////////////


#include<stdio.h>
#include<stdlib.h>

int SumEven(int Arr[] , int iSize)
{
    int iCount = 0 , iSum = 0 ;

    for(iCount = 0 ; iCount < iSize ; iCount++)
    {
        if((Arr[iCount] % 2) == 0)
        {
            iSum = iSum + Arr[iCount];
        }
    }
    return iSum;
}


int main()
{
    int iLength = 0 , iCount = 0 , iRet = 0 ;
    int *ptr = NULL;    

    printf("Enter Number of Elements : ");
    scanf("%d" , &iLength);

    ptr = (int *)malloc(iLength * (sizeof(int)));

    if(ptr == NULL)
    {
        printf("Unable to allocate Memory\n");
        return -1;
    }

    printf("Enter the Elements :\n");

    for(iCount = 0 ; iCount < iLength ; iCount++)
    {
        scanf("%d" , &ptr[iCount]);
    }

    iRet = SumEven(ptr , iLength);

    printf("Summation of Even Numbers : %d" , iRet);

    free(ptr);
}

File: JAVA/Program44.java
//////////////////////////////////////////////////////////////////////////////////////////
//                                                                                      //
//  File name :     Program44.java                                                      //
//  Description :   Used to Print Given Pattern                                         //
//  Author :        Pallavi Omprakash Malewar                                           //
//  Date :          26/08/2025                                                          //
//                                                                                      //
//////////////////////////////////////////////////////////////////////////////////////////


// Input : 4    4
// Output :
/*
    o   *   *   *
    *   o   *   *
    *   *   o   *
    *   *   *   o
*/
            

import java.util.Scanner;

class Pattern
{
    public void Display(int iRow , int iCol)
    {
        int i = 0, j = 0;

        for(i = 1 ; i <= iRow ; i++)
        {
            for(j = 1 ; j <= iCol ; j++)
            {
                if(i == j)
                {
                    System.out.print("o\t");
                }
                else
                {
                    System.out.print("*\t");
                }
            }
            System.out.println();
        }
    }
}

class Program44
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.print("Enter Number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter Number of Columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}